pyproject.toml

```diff
@@ -9,13 +9,13 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "~3.8"
+python = ">=3.8"
```

Comment:
why this change? 
if it's above 3.11, then it's not compatible with torch. 
maybe add also an upper bound?

Reply:
Because otherwise we cannot have 3.10 in OOR. Adding the upper bound.